src/components/sbb-button/sbb-button.stories.tsx

```diff
@@ -563,6 +570,15 @@ export const RequestSubmit: StoryObj = {
   },
 };
 
+export const LoadingIndicator: StoryObj = {
+  render: LoadingIndicatorTemplate,
+  argTypes: defaultArgTypes,
+  args: {
+    ...defaultArgs,
+    variant: variant.options[1],
+  },
```

Comment:
button has to be disabled during loading

Reply:
Done in [4a599f5](https://github.com/lyne-design-system/lyne-components/pull/1968/commits/4a599f567fc947da505baa48cf46bef3eb3455d6)